spinach/__init__.py

```diff
@@ -1,8 +1,5 @@
 from .brokers.memory import MemoryBroker
 from .brokers.redis import RedisBroker
-from .const import VERSION
 from .engine import Engine
 from .task import Tasks, Batch, RetryException, AbortException
 from .worker import ThreadWorkers, AsyncioWorkers
-
-__version__ = VERSION
```

Comment:
Hmm...  This looks like it could be a breaking change, especially if downstream consumers like to inspect `spinach.__version__`.

Fixing...